extensions/git/src/commands.ts

```diff
@@ -2606,6 +2607,24 @@ export class CommandCenter {
 		await repository.branch(branchName, true, target);
 	}
 
+	private async _unsortedQuickPick<T extends QuickPickItem>(items: Promise<T[]>, placeHolder: string): Promise<T | undefined> {
```

Comment:
Can we rename this to `pickRef()` as we already have a method called `pickStash()` that is similar.

Reply:
Done